### ISSUE TITLE
Use token-protected API route for project page, close public API role 

### ISSUE THREAD

**Issue by friep** [CorrelAid/correlaid_website#468]
use token protected API route to get data for

- projects
- organizations
- project people 
- project outputs 

then close the public API role for those collections. 

follow up to #457

**Comment by KonradUdoHannes** [comment]
We have a new directus role (`website builder`) with reade access to the mentioned (and some related collections). We also switched the project related website functionality to use an authenticated fetch (#480 ) to authenticate as a user with that role.

No permissions where removed yet from the public role.

I wonder whether the project related read access of the public API role should be removed, or whether we might eventually want to add configuration options to build the website with public projects only. Even if we don't want to deploy such a configuration, it would allow contributors without directus access to build the entire site locally for testing etc. So the question boils down to whether we expect contributors that should ideally not have access to all the raw (non-anonymized) project data.

**Comment by friep** [comment]
that's a good point, @KonradUdoHannes ! thanks for raising it. It's indeed nice to offer the opportunity to build the website without the token for potential collaborators (e.g. who want to fix a bug quickly etc).

I went through the project related collections and made sure that we share only content that we want to be public via the **public role**. 👍 The main filter is `project.status == 'published'` , this is used in all related collections. 
the dependency on the status of the organization was removed. Is this field still used in the frontend code? If not, I'll remove it.
